tools/o3de-doxygen/README.md

```diff
@@ -0,0 +1,184 @@
+# C++ API Reference Generation
+
+The [Open 3D Engine (O3DE) API Reference](https://www.o3de.org/docs/api/) is generated using [Doxygen](https://www.doxygen.nl/). With the help of the API Reference Generation (`o3de-doxygen`) scripts to automate the process, an API reference is generated for all frameworks and Gems in the [`o3de` repository](https://github.com/o3de/o3de.org) by default.  You may also configure the scripts to [generate API references for Gems in other O3DE repositories]((#generating-for-gems-outside-of-o3de)).
+
+
+## Requirements
+
+* Linux machine or Windows Subsystem for Linux (WSL)
+* Clone of the `o3de` repository in your machine
+* Clone of the `o3de.org` repository in your machine
+* [Doxygen](https://www.doxygen.nl/manual/install.html) installation
+
+## Set up 
+
+### Download `o3de-doxygen`
+
+1. Clone the  [`sig-docs-community` repository](https://github.com/o3de/sig-docs-community/), which contains the `o3de-doxygen` scripts.
+2. In the `tools` directory, find the `o3de-doxygen` scripts.
+
+
+### Install rust commands
+
+The `o3de-doxygen` scripts use some efficient Rust commands for processing the source code in `o3de`. You must install Rust and the `fd-find` cargo to run the scripts properly. 
+
+
+1. Install Rust by using [rustup](https://rustup.rs/), or by running the following command in your Linux machine: 
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    ```
+
+1. Add rust to your environment variables.
+
+    ```shell
+    source "$HOME/.cargo/env"
+    ```
+
+1. Install the `fd-find` cargo, which is a file searching tool:
+
+    ```shell
+    cargo install fd-find
+    ```
+
+
+
+### Clean repos
+
+The C++ API reference generation scripts runs Doxygen on your local `o3de` clone. Then it outputs the generated files into your local `o3de.org` clone. Therefore, before you generate the API, you must ensure that your cloned repos are clean and have the latest changes.
```

Comment:
noun/verb agreement
```suggestion
The C++ API reference generation script runs Doxygen on your local `o3de` clone. Then it outputs the generated files into your local `o3de.org` clone. Therefore, before you generate the API, you must ensure that your cloned repos are clean and have the latest changes.
```

Reply:
```suggestion
The C++ API reference generation scripts run Doxygen on your local `o3de` clone. Then it outputs the generated files into your local `o3de.org` clone. Therefore, before you generate the API, you must ensure that your cloned repos are clean and have the latest changes.
```

Reply:
In that case, I'd recommend changing the second sentence to match.